src/BTree.h

```diff
@@ -222,6 +220,14 @@ struct default_strategy<int> : public linear {};
 template <typename... Ts>
 struct default_strategy<std::tuple<Ts...>> : public linear {};
 
+/**
+ * The default non-updater
+ */
+template <typename T>
+struct updater {
+    void update(T& old_t, const T& new_t) {}
```

Comment:
If you're not using parameters, don't name them. You can still use commented names to help auto-complete/doc helps. e.g. `/* old_t */`